portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,57 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function randomizeImage() {
+  // The images directory contains 13 images, so generate a random index between
+  // 1 and 13.
+  const imageIndex = Math.floor(Math.random() * 5) + 1;
+  const imgUrl = 'images/jason-' + imageIndex + '.jpg';
+
+  const imgElement = document.createElement('img');
+  imgElement.src = imgUrl;
+
+  const imageContainer = document.getElementById('random-image-container');
+  // Remove the previous image.
+  imageContainer.innerHTML = '';
+  imageContainer.appendChild(imgElement);
+}
+
+function modalbtn() {
+    var button = document.getElementById("Intro");
+    var modal = document.getElementById("myModal");
+    modal.style.display = "block";
+}
+
+function modalbtn2() {
+    var button = document.getElementById("Interests");
+    var modal = document.getElementById("myModal2");
+    modal.style.display = "block";
+}
+
+function spanbtn(){
+    var span = document.getElementsByClassName("close")[0];
+    var modal = document.getElementById("myModal");
+    modal.style.display = "none";
+}
+
+function spanbtn2(){
+    var span = document.getElementsByClassName("close")[0];
+    var modal = document.getElementById("myModal2");
+    modal.style.display = "none";
+}
```

Comment:
where are these functions used? and what does modalbtn and spanbtn functions do?